Source/JavaScriptCore/runtime/SymbolPrototype.cpp

```diff
@@ -110,7 +110,13 @@ JSC_DEFINE_HOST_FUNCTION(symbolProtoFuncToString, (JSGlobalObject* globalObject,
     if (!symbol)
         return throwVMTypeError(globalObject, scope, SymbolToStringTypeError);
     Integrity::auditStructureID(symbol->structureID());
-    RELEASE_AND_RETURN(scope, JSValue::encode(jsNontrivialString(vm, symbol->descriptiveString())));
+    auto description = symbol->tryGetDescriptiveString();
+    if (!description) [[unlikely]] {
+        ASSERT(description.error() == ErrorTypeWithExtension::OutOfMemoryError);
+        throwOutOfMemoryError(globalObject, scope);
+        return encodedJSValue();
```

Comment:
For new code, let's just use `return { };`.

Reply:
Done.